benchmark/fluid/A2C/README.md

```diff
@@ -14,15 +19,15 @@ Performance of A2C on some envrionments in training process after 10 million sam
 
 ## How to use
 ### Dependencies
-+ [paddlepaddle>=2.0.0](https://github.com/PaddlePaddle/Paddle)
-+ [parl>=1.4.3](https://github.com/PaddlePaddle/PARL)
++ [paddlepaddle>=1.8.5](https://github.com/PaddlePaddle/Paddle)
```

Comment:
For the examples of fluid, the requirement of paddlepaddle should be <2.0.0?

Reply:
fixed